libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/ITwilioAPI.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Adapters.Twilio
+{
+    public interface ITwilioApi
+    {
+        void LogIn(string username, string password);
+
+        Task<object> CreateMessageResourceAsync(object messageOptions);
```

Comment:
Here we should replace the type object with MessageResource and CreateMessageOptions respectively.

Reply:
I didnt want to use an specific type from the Twilio library to make it generic.
If we had to use other API, we would need to change the interface.

I did additional changes, we can discuss alternatives.